config/connection.js

```diff
@@ -7,9 +7,9 @@ const connection = mysql.createConnection({
     host: "localhost",
     port: 3306,
     user: "root",
-
+    password: "Root1234!",
```

Comment:
don't wanna store your passwords in git, use the password: process.env.PASSWORD code